src/Common/ConfigurationUtility.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Configuration;
+
+namespace Microsoft.Azure.WebJobs.Extensions
+{
+    internal static class ConfigurationUtility
+    {
+        // This method is taken from: https://github.com/Azure/azure-webjobs-sdk/blob/master/src/Microsoft.Azure.WebJobs.Host/ConfigurationUtility.cs
+        public static string GetSettingFromConfigOrEnvironment(string settingName)
+        {
+            string configValue = ConfigurationManager.AppSettings[settingName];
+            if (!string.IsNullOrEmpty(configValue))
+            {
+                // config values take precedence over environment values
+                return configValue;
+            }
+
+            return Environment.GetEnvironmentVariable(settingName) ?? configValue;
+        }
+    }
+}
```

Comment:
I'm going to undo this -- all these extensions already have some internal Resolve() method. I'll use that for now and we can expose one from core in the future.